.github/workflows/ci.yml

```diff
@@ -111,14 +111,9 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [14, 16, 18, 20]
+        node-version: [18, 20, 22]
         os: [macos-latest, ubuntu-latest, windows-latest]
         exclude:
-          # excludes node 14 on Windows
-          - os: windows-latest
-            node-version: 14
-          - os: macos-latest
-            node-version: 14
           - os: macos-latest
             node-version: 16
```

Comment:
Not that important but node 14 exclusions were removed so why not also remove 16?

```suggestion
```